portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,31 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.gson.Gson;
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> messages = new ArrayList<>();
+
+  public DataServlet() {
+    super();
+    messages.add("message 1");
```

Comment:
Consider extracting the message initialization into a helper method like "initializeMessages()". If your constructor grows in size, it will be easy to keep track of what is going on. And it improves the testability of your code, as it's easier to test methods responsible for small individual tasks.

Reply:
This makes sense. Will change.